Apps/Mock/src/Controllers/PhsaController.cs

```diff
@@ -36,14 +36,29 @@ public class PhsaController : ControllerBase
         /// <param name="limit">The limit on the number of records returned.</param>
         /// <returns>The mocked laboratory order json.</returns>
         [HttpGet]
-        [Route("laboratoryOrder")]
+        [Route("LaboratoryOrders")]
```

Comment:
We also need to override the endpoint configured in appsettings.hgmock.json as "PlisLabEndPoint".

For the routes, we should set them in accordance with the values configured in the appsettings to return what the delegate expects.

The delegate expects to retrieve the laboratory results from here:
`$"{this.labConfig.BaseUrl}{this.labConfig.PlisLabEndPoint}/LabSummary"`

And the COVID-19 laboratory results from here:
`$"{this.labConfig.BaseUrl}{this.labConfig.LabOrdersEndpoint}"`